docs/reference/nodes/workflow.md

```diff
@@ -0,0 +1,44 @@
+# Workflow Nodes
+
+
+## Input & Output Nodes
+*Input* and *Output* nodes are used to define the required inputs and outputs of a workflow to be executed externally. Setting these nodes is essential to be able to nest workflows or executing the workflow in [batch mode](../../getting_started/basics/batch.md). The custom plugs from these nodes, which should be of type `through`, will act as data pathways, linking information set externally in the workflow to the current workflow and viceversa. 
+
+
+![Workflow Node](../../images/nodes/Input_outputNodes.png) 
+
+## Workflow Node
+The *Workflow* node executes an external workflow different from the workflow the node is located in, allowing, this way, to nest workflows. The node expands its plugs dynamically to enable access to the referenced workflow input and output data.
+
+![Workflow Node](../../images/nodes/workflow.png) 
+
+### Inputs
+
+- **File**: This plug of type [FileIn](../nodes/#plugs) is used to reference the workflow file (*.sft* extension) that will be executed by the node. When the `file` plug content is set, the node will automatically update by creating the input and output plugs corresponding to the sourced workflow inputs and outputs.
+
+## WorkflowProcess Node
+
+The *WorkflowProcess* node works similarly to the *Workflow* node, however, the execution of the referenced workflow will be run by an external process. By default, the workflow will be executed in a new instance of Python, however, the node allows to choose Python interpreter the workflow will run in. 
+
+
+![WorkflowProcess Node](../../images/nodes/workflow_process.png)
+
+The available Python interpreters will depend on the environment variable configuration. To allow the *WorkflowProcess* find new Python interpreters, it must be added a new variable to the environment with the following format `SHIFT_PROCESS_<NAME OF THE INTERPRETER>`. The value of this variable must be the path to the desired Python interpreter.

```

Comment:
```suggestion
The available Python interpreters will depend on the environment configuration. To allow the *WorkflowProcess* to find new Python interpreters, a new environment variable must be set with the following format `SHIFT_PROCESS_<NAME OF THE INTERPRETER>`. The value of this variable must be the path to the desired Python interpreter (.exe file for Windows users).

```

Reply:
Almost done. I'm waiting to know if we should make specific mentions to windows or not.

Reply:
It is specified in the example for maya, not the general description